components/alttextify/actions/submit-image/submit-image.mjs

```diff
@@ -0,0 +1,112 @@
+import fs from "fs";
+import alttextify from "../../alttextify.app.mjs";
+import { checkTmp } from "../../common/utils.mjs";
+
+export default {
+  key: "alttextify-submit-image",
+  name: "Submit Image to Alttextify",
+  description: "Upload or submit an image to Alttextify for alt text generation. [See the documentation](https://apidoc.alttextify.net/#api-Image-UploadRawImage)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    alttextify,
+    alert: {
+      type: "alert",
+      alertType: "info",
+      content: "Supported formats: JPEG, PNG, GIF, WEBP, BMP\nMaximum file size: 16 MB\nMinimum dimensions: 50 x 50 (smaller images may not be able to generate alt text).",
+    },
+    async: {
+      propDefinition: [
+        alttextify,
+        "async",
+      ],
+    },
+    image: {
+      type: "string",
+      label: "Image Path",
+      description: "The path to the image file in the `/tmp` directory. [See the documentation on working with files](https://pipedream.com/docs/code/nodejs/working-with-files/#writing-a-file-to-tmp)",
+    },
```

Comment:
We should use the new `getFileStream` for uploading an image. This allows the user to enter either a URL or a filePath. 
[Slack convo](https://pipedream-users.slack.com/archives/G01J50YGJGP/p1749054362633079)
[Issue #16977](https://github.com/PipedreamHQ/pipedream/issues/16977)